src/nnet3/natural-gradient-online.h

```diff
@@ -375,7 +375,7 @@ namespace nnet3 {
    * Initialization *
 
    Now, a note on what we do on time t = 0, i.e. for the first minibatch.  We
-   initialize X_0 to the top R eigenvectors of 1/N X_0 X_0^T, where N is the
+   initialize R_0 to the top R eigenvectors of 1/N X_0 X_0^T, where N is the
```

Comment:
I think there might be another problem here.  I believe X is (N by D) where N is the minibatch size.  (Although elsewhere I seem to have used M for the minibatch size so that might be a better letter).  We want the eigenvectors of a D x D matrix, so it should be X_0^T X_0.    That appears in the lines immediately below, too.